SpatialGDK/Source/SpatialGDKEditor/Public/SpatialOSLoadBalancingStrategies.h

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "UObject/Object.h"
+#include "Serialization/JsonWriter.h"
+
+#include "SpatialOSLoadBalancingStrategies.generated.h"
+
+UCLASS(Abstract)
+class SPATIALGDKEDITOR_API USpatialOSAbstractLoadBalancingStrategy : public UObject
```

Comment:
Why the USpatialOS* naming? If it's the need to differentiate from the actual lb class then maybe something like the word Config or Settings?

Reply:
Naming is hard. I want to convey that this LoadBalancing Strategy is matched against what SpatialOS provides. As opposed to what you can implement in the GDK.

Reply:
Call it *Runtime* then? SpatialOS is everything together

Reply:
That would work.